lean-toolchain

```diff
@@ -0,0 +1 @@
+leanprover/lean4:nightly
```

Comment:
There are no .lean files in this repo anyway, are there?

Reply:
Oh, it is for `mdbook test`.

Reply:
Let's use a pinned release then and bump it when we fix breakage from upstream. Tests should be deterministic.

Reply:
Ok, I pinned it to leanprover/lean4:nightly-2022-02-01